Src/DeUrgenta.Backpack.Api/Commands/AddBackpackItem.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using CSharpFunctionalExtensions;
+using DeUrgenta.Backpack.Api.Models;
+using MediatR;
+
+namespace DeUrgenta.Backpack.Api.Commands
+{
+    public class AddBackpackItem : IRequest<Result<BackpackItemModel>>
```

Comment:
It is recommended to make commands/queries immutable . Please remove `set;` from all properties in all commands/queries.